### DISCUSSION TITLE
how can I change the user?

### DISCUSSION THREAD

**Discussion by JustFrederik** [zed-industries/zed#10633] [Feature Requests]
I want to log out and log into another account. If I log out I login to the account I was logged into before. I changed gh auth login & git config --global

**Comment by SomeoneToIgnore** [comment]
Unfortunately, it's been an issue for quite a while: https://github.com/zed-industries/zed/issues/4716

https://github.com/zed-industries/zed/pull/10558 aims to fix it and gets to release in Zed Preview today.
